examples/tx-scheduler/deploy-contract/app.ts

```diff
@@ -0,0 +1,27 @@
+// tslint:disable: no-console
+
+import * as util from 'util'
+
+import { TxSchedulerTransactions } from '../../../src/http/tx-scheduler'
```

Comment:
I think we can call this class `Transactions` or `TxProducer`

Reply:
`Transaction` sounds very generic, maybe `TxSchedulerClient` ??